WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductCoordinator.swift

```diff
@@ -102,6 +103,11 @@ final class AddProductCoordinator: Coordinator {
         ServiceLocator.analytics.track(event: .ProductCreation.addProductStarted(source: source,
                                                                                  storeHasProducts: storeHasProducts))
 
+        // TODO-10688: Replace this with eligibility check
+        let isEligibleForAI = ServiceLocator.featureFlagService.isFeatureFlagEnabled(.productCreationAI)
+        guard !isEligibleForAI else {
+            return presentActionSheetWithAI()
```

Comment:
❓ 

Should we present the AI sheet after checking the following `shouldSkipBottomSheet()` condition? 

I believe presenting without checking `shouldSkipBottomSheet()` will prevent the AI announcement modal from presenting the product form directly.

Reply:
Good catch! I add that check in 600074df7ed97ccad29080cb8a00ce77606e7114.